src/v/cloud_roles/refresh_credentials.cc

```diff
@@ -365,7 +366,8 @@ ss::future<> refresh_credentials::impl::init_tls_certs() {
         co_await b.set_system_trust();
     }
 
-    _tls_certs = co_await b.build_reloadable_certificate_credentials();
+    _tls_certs = co_await net::build_reloadable_credentials_with_probe<
+      ss::tls::certificate_credentials>(std::move(b), "api_client", name);
```

Comment:
Is "api_client" sufficiently descriptive and unique?

Reply:
No... perhaps "cloud_provider_client"